msbuild/Xamarin.Mac.Tasks/Tasks/Metal.cs

```diff
@@ -1,13 +1,18 @@
 ﻿using System.IO;
 
+using Xamarin.MacDev;
 using Xamarin.MacDev.Tasks;
 
 namespace Xamarin.Mac.Tasks
 {
 	public class Metal : MetalTaskBase
 	{
 		protected override string OperatingSystem {
-			get { return "osx"; }
+			get { return "macosx"; }
+		}
+
+		protected override string MinimumDeploymentTargetKey {
+			get { return ManifestKeys.LSMinimumSystemVersion; }
```

Comment:
My previous comment/question remains :)
It's not a `Base` type so it should be empty right ?

Reply:
There's already partial support for XM, how does it work ?
@emaf ?

Reply:
@spouliot, the partial support for XM we have on VS is just letting the users open the project and building it just on Windows, so we're not running tasks remotely on the Mac.

So for now this change is fine, and in the future if we want to support full builds for XM on VS we'll need to fix that using a base task on Xamarin.Mac.Tasks.Core. As Jeff mentioned, XM tasks are not following this appoach right now.

Reply:
@emaf thanks for clarifying
@jstedfast we can postpone that - it's unrelated, but `\Library/Frameworks/Mono.framework/External/xbuild/Xamarin/Mac/Xamarin.Mac.Tasks.Core.dll` does exists :)